.github/ISSUE_TEMPLATE/airflow_helmchart_bug_report.yml

```diff
@@ -28,7 +28,7 @@ body:
         What Apache Airflow Helm Chart version are you using?
       multiple: false
       options:
-        - "2.3.0b1 (pre-release)"
+        - "2.3.0rc1 (pre-release)"
```

Comment:
Why it's here to begin with? The question is about helm chart version not Airflow version
There is another question later "Apache Airflow version"

Reply:
Fixed.